priv/ie_gen_v1.erl

```diff
@@ -366,15 +365,14 @@ ies() ->
      {192, "Evolved Allocation/Retention Priority II", '_',
       []},
      {193, "Extended Common Flags", '_',
-      [{"Flags", {flags, ["Unauthenticated IMSI",
-			  "CCRSI",
-			  "CPSR",
-			  "RetLoc",
-			  "VB",
-			  "PCRI",
+      [{"Flags", {flags, ["UASI",
```

Comment:
Please comment about change of order.
It is reversed, but why?
Any order (that is not alphabetical) is easier to understand with a comment.

Reply:
A comment saying that the order is according to 3GPP TS would be good.
To avoid that somebody change the order when editing the code.